src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/DevAppServerArgs.java

```diff
@@ -71,4 +72,12 @@
     }
     return Collections.emptyList();
   }
+
+  /**
+   * @return {@code [--name, key1=val1, --name, key2=val2, ...]} or {@code []} if
+   *      keyValues=empty/null.
+   */
+  public static List<String> get(String name, Map<String, String> keyValues) {
+    return Args.namedKeyValues(name, keyValues);
```

Comment:
fair point. I'm wondering if namedKeyValues is too generic. At first read I think 'what's the difference between a key and a name?'. What do you think of something like flagPerEntry(flagName, map)